libs/dh/wholesale/feature-settlement-reports-v2/src/lib/table/dh-settlement-reports-table.component.html

```diff
@@ -55,7 +55,11 @@
     </ng-container>
 
     <ng-container *wattTableCell="columns['status']; header: t('columns.status'); let entry">
-      <dh-settlement-reports-status [status]="entry.statusType" />
+      <dh-settlement-reports-status
+        [status]="entry.statusType"
+        [reportId]="entry.id"
+        (download)="downloadReport($event)"
+      />
```

Comment:
suggestion: `reportId` is not doing anything special in `dh-settlement-reports-status` so it can be skipped. Instead, it can be directly sent to `downloadReport()`, e.g.

```suggestion
      <dh-settlement-reports-status
        [status]="entry.statusType"
        (download)="downloadReport(entry.id)"
      />
```

The click event in `dh-settlement-reports-status` can then just emit.

Reply:
Good idea, looks better :100: